### PR TITLE
Translate to Chinese: Chap 1

### PR DESCRIPTION
**Pull request by daming-lu**

### CODE REVIEW

File: 01.fit_a_line/README.cn.md [hunk 1]
```diff
@@ -110,157 +94,158 @@ import paddle.v2.dataset.uci_housing as uci_housing
...
-parameters = paddle.parameters.create(cost)
+test_reader = paddle.batch(
+    paddle.reader.shuffle(
+        paddle.dataset.uci_housing.test(), buf_size=500),
+    batch_size=BATCH_SIZE)
 ```
 
-### 创建Trainer
+### 配置训练程序
+训练程序的目的是定义一个训练模型的网络结构.对于线性回归来讲，它就是一个从输入到输出的简单的全连接层。更加复杂的结果，比如卷积神经网络，递归神经网络等会在随后的章节中介绍。训练程序必须返回`平均损失`作为第一个返回值，因为它会被后面反向传播算法所用到。
```
Comment (jetfuel) [inline 01.fit_a_line/README.cn.md:129]: 网络结构.对于线性
Incorrect period.

File: 01.fit_a_line/README.cn.md [hunk 2]
```diff
@@ -110,157 +94,158 @@ import paddle.v2.dataset.uci_housing as uci_housing
...
+
+### 创建训练器
+训练器会读入一个训练程序和一些必要的其他参数：
 
 ```python
-feeding={'x': 0, 'y': 1}
+trainer = fluid.Trainer(
+    train_func=train_program,
+    place=place,
+    optimizer=fluid.optimizer.SGD(learning_rate=0.001))
```
Comment (jetfuel) [inline 01.fit_a_line/README.cn.md]: Will need to use the optimizer_func later.

File: 01.fit_a_line/README.cn.md [hunk 3]
```diff
@@ -110,157 +94,158 @@ import paddle.v2.dataset.uci_housing as uci_housing
...
+    num_epochs=100,
     event_handler=event_handler_plot,
-    num_passes=30)
+    feed_order=feed_order)
 ```
 
 ![png](./image/train_and_test.png)
 
-### 应用模型
+## 推测
```
Comment (jetfuel) [inline 01.fit_a_line/README.cn.md]: Maybe we should use '预测', although I honestly don't know if that's the go to translation.

**Review by jetfuel — APPROVED** [review]
Otherwise LGTM

File: 01.fit_a_line/README.cn.md (hunk 2 above)
Comment (daming-lu) [reply 01.fit_a_line/README.cn.md]: Good catch

File: 01.fit_a_line/README.cn.md (hunk 1 above)
Comment (daming-lu) [reply 01.fit_a_line/README.cn.md:129]: missed this. I will fix it in the next PR on Chap 4

**Pull request merged**